### PR TITLE
Adds 5 new backpacks

### LINKED ISSUE #32332: Available backpacks are too small
IRL backpacks come with volumes of 5-120L or more, while the backpacks used by tourists in autonomous settings are 70L+, but the biggest backpack available in cdda is 30L, making real backpacker experience unworkable in cdda without incurring crippling penalties of wearing multiple shitty backpacks at once.

modern 80L backpack

### PR DESCRIPTION
**Pull request by Hymore246**
#### Summary

```SUMMARY: Content "Adds 5 new backpacks"```

#### Purpose of change

Fixes #32332
Adds 5 new backpacks of various sizes to the game.

#### Describe the solution

This change adds 5 new backpacks and recipes, Below are the details for the new backpacks:

Name | Encumbrance | Storage (L) | Special
--- | --- | --- | --- 
high-volume rucksack|60|100.00
hiking pack|35|55.00|Activate to sheath up to 2 small knifes/swords
large tactical backpack|45|65.00|
petpack|14|3.75|Activated to store a small creature (cat, chicken, etc) inside just like a pet carrier
travelpack|14|25.00|
#### Describe alternatives you've considered

I think the encumbrance and storage are balanced but I'm open to other opinions. As far as I know, encumbrance as a whole needs an overhaul but I'll adjust the items as necessary.

#### Additional context

I did not come up with the idea for these backpacks. The user "ghost" created these backpack in #32426. However, ghost has delete their account and closed the PR for some unknown reason. I did not want to see a good idea go to waste, so I'm adding them in myself.

### CODE REVIEW

**Comment by Hymore246** [comment]
Jenkins rebuild

**Comment by LaVeyanFiend** [comment]
As I know @anothersimulacrum will request once he sees this, can you change the formatting of their volumes to the milliliter/liter format?

File: data/json/items/armor.json [hunk 1]
```diff
@@ -10210,5 +10210,112 @@
...
+    },
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "petpack",
+    "type": "ARMOR",
+    "name": "petpack",
+    "description": "Before the Cataclysm this would allow your four-legged friend to see the world, now it's used to shield them from the world.",
+    "weight": 1316,
+    "volume": 40,
```
Comment (LaVeyanFiend) [inline data/json/items/armor.json]:
```suggestion
    "volume": "10 L",
```
Example

Comment (Hymore246) [reply data/json/items/armor.json]: That might explain why jenkins is freaking out.

File: data/json/items/armor.json [hunk 2]
```diff
@@ -10210,5 +10210,112 @@
...
+    "material_thickness": 6,
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "travelpack",
+    "type": "ARMOR",
+    "name": "travelpack",
+    "description": "A hiking pack used for short trips.",
+    "weight": 636,
+    "volume": 12,
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]:
```suggestion
    "volume": "3 L",
```

File: data/json/items/armor.json [hunk 3]
```diff
@@ -10210,5 +10210,112 @@
...
+    "use_action": "CAPTURE_MONSTER_ACT",
+    "flags": [ "BELTED", "WATERPROOF" ]
+  },
+  {
+    "id": "backpack_tactical_large",
+    "type": "ARMOR",
+    "name": "large tactical backpack",
+    "description": "A large tactical multi-compartment backpack that has great lower back support.",
+    "weight": 2291,
+    "volume": 60,
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]:
```suggestion
    "volume": "15 L",
```

File: data/json/items/armor.json (hunk 1 above)
Comment (anothersimulacrum) [inline data/json/items/armor.json]:
```suggestion
    "volume": 10 L",
```

File: data/json/items/armor.json [hunk 4]
```diff
@@ -10210,5 +10210,112 @@
...
+    "material_thickness": 6,
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "backpack_hiking",
+    "type": "ARMOR",
+    "name": "hiking backpack",
+    "description": "A large sized hiking backpack with plenty of storage space.",
+    "weight": 2291,
+    "volume": 55,
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]: Rounding up a bit.
```suggestion
    "volume": "14 L",
```
No change.
```suggestion
    "volume": "13750 ml",
```

File: data/json/items/armor.json [hunk 5]
```diff
@@ -10210,5 +10210,112 @@
...
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "high-volume rucksack",
+    "description": "Also know as \"a bigpack\", this rucksack is prized by pack rats but may lead to a bad back.",
+    "weight": 3539,
+    "volume": 85,
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]: Rounding up a bit.
```suggestion
    "volume": "21 L",
```
True to the original
```suggestion
    "volume": "21250 ml",
```

**Comment by ampersand55** [comment]
I think the current backpacks need to be rebalanced in relation to these.

E.g. the travelpack has the same storage as the survivor rucksack (25L), but is much better.

The survivor rucksack is higher encumbrance, lower armor, higher coverage, higher skill requirements requires more materials to craft, not waterproof, heavier and takes up more space.

The normal backpack, the military rucksack and the duffelbag seem to be able to store encumbrance x 4 units of 250ml.

The survivor backpack and the survivor rucksack seem to be able to store encumbrance x 5 + 10 units of 250ml, and the survivor duffelbag is a little better.

A table:

| Item | Current encumbrance | Normal encumbrance | High-end "survivor" encumbrance  | Storage
| --- | --- | --- | --- | --- |
| high-volume rucksack | 60 | 100 | 78 | 100L |
| hiking pack | 35 | 55 | 42 | 55L |
| large tactical backpack | 45 | 65 | 50 | 65L |
| travelpack | 14 | 25 | 18 | 25L |
| backpack | 10 |10 | 6 | 10L |
| military rucksack | 20 | 20 | 14 | 20L |
| duffel bag | 30 | 30 | 22 | 30L |
| survivor backpack | 8 | 12.5 | 8 | 12.5L |
| survivor rucksack | 18 | 25 | 18 | 25L |
| survivor duffelbag | 25 | 37.5 | (28) | 37.5L |

File: data/json/items/armor.json [hunk 6]
```diff
@@ -10210,5 +10210,112 @@
...
+    "id": "travelpack",
+    "type": "ARMOR",
+    "name": "travelpack",
+    "description": "A hiking pack used for short trips.",
+    "weight": 636,
+    "volume": 12,
+    "price": 7000,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "color": "orange",
```
Comment (anothersimulacrum) [inline data/json/items/armor.json]: This isn't a color.

Comment (Hymore246) [reply data/json/items/armor.json]: I'll just change it to yellow.

**Comment by Hymore246** [comment]
> I think the current backpacks need to be rebalanced in relation to these.

I agree and there should probably be a few more containers added too. But that's outside the scope of this PR.

**Comment by ampersand55** [comment]
> 
> 
> > I think the current backpacks need to be rebalanced in relation to these.
> 
> I agree and there should probably be a few more containers added too. But that's outside the scope of this PR.

Adjusting the encumbrance to be the same as the storage in L would be enough to make them consistent with the current backpacks and imho mergable.

**Comment by Hymore246** [comment]
> 
> 
> > > I think the current backpacks need to be rebalanced in relation to these.
> > 
> > 
> > I agree and there should probably be a few more containers added too. But that's outside the scope of this PR.
> 
> Adjusting the encumbrance to be the same as the storage in L would be enough to make them consistent with the current backpacks and imho mergable.

Yeah, that seems fair. Most containers seem to follow that trend. There should probably be another PR in the future to rethink the standard formula.

File: data/json/items/armor.json [hunk 7]
```diff
@@ -10210,5 +10210,112 @@
...
+    "coverage": 75,
+    "encumbrance": 55,
+    "storage": 220,
+    "warmth": 6,
+    "material_thickness": 6,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Sheath blade",
+      "holster_msg": "You sheath your %s",
+      "min_volume": 1,
```
Comment (LaVeyanFiend) [inline data/json/items/armor.json]:
```suggestion
      "min_volume": "250ml",
```
I'm honestly not sure if min_volume and max_volume accept the milliliter/liter format.

**Comment by Hymore246** [comment]
Jenkins rebuild

**Comment by l29ah** [comment]
Re encumbrance: https://github.com/CleverRaven/Cataclysm-DDA/issues/32332#issuecomment-511118817

**Comment by Hymore246** [comment]
Jenkins rebuild

File: data/json/items/armor.json [hunk 8]
```diff
@@ -10210,5 +10210,112 @@
...
+    "encumbrance": 55,
+    "storage": 220,
+    "warmth": 6,
+    "material_thickness": 6,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Sheath blade",
+      "holster_msg": "You sheath your %s",
+      "min_volume": "250ml",
+      "max_volume": 10,
```
Comment (LaVeyanFiend) [inline data/json/items/armor.json]:
```suggestion
      "max_volume": "2500ml",
```

**Comment by Hymore246** [comment]
Jenkins rebuild

**Comment by Hymore246** [comment]
Travis rebuild

**Pull request merged**